### PR TITLE
Build draft CUBE landing page

### LINKED ISSUE #8825: Draft build CUBE landing page from wireframe
Build template for CUBE landing page, hard coded based on this [wire frame](https://app.zeplin.io/project/5f5268731d98b38800528395/screen/5fa2d5661e336e12728b56a3)

### PR DESCRIPTION
**Pull request by bethcollins92**
## Done

- Hard code CUBE landing page from wireframe
- Design hasn't been done yet so this is a draft only
- Add secondary nav for cube bubble

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/cube
- Check against [wireframe](https://app.zeplin.io/project/5f5268731d98b38800528395/screen/5fa2d5661e336e12728b56a3)

## Issue / Card

Fixes #8825

### CODE REVIEW

File: webapp/views.py [hunk 1]
```diff
@@ -1229,3 +1229,9 @@ def cube_microcerts():
     }
 
     return flask.render_template("cube/microcerts.html", **data)
+
+
+def cube_home():
+    data = {}
```
Comment (jpmartinspt) [inline webapp/views.py:1235]: You can remove it from here and from the function call bellow, unless you will need this in the future?

Comment (bethcollins92) [reply webapp/views.py:1235]: I was thinking I will need the badge icon data in the future, but I could always just hard code them from the asset server now I think about it

Comment (jpmartinspt) [reply webapp/views.py:1235]: Has you prefer, I was just wondering, happy to keep reviewing without changes.

Comment (bethcollins92) [reply webapp/views.py:1235]: Ok thanks, I'll leave it there for now and then remove it if I end up not needing it

**Comment by jpmartinspt** [comment]
Code looks good, I'll let @anasereijo's eagle eye QA :smile:

**Comment by bethcollins92** [comment]
QA label removed as only draft design for now, and links are not live. Page will have design, QA and code review on next PR.

**Pull request merged**